aws/resource_aws_ecr_repository_test.go

```diff
@@ -2,15 +2,64 @@ package aws
 
 import (
 	"fmt"
+	"log"
 	"testing"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ecr"
+	multierror "github.com/hashicorp/go-multierror"
```

Comment:
No need to have `multierror` as an alias as that's now the package name inside `go-multierror`.